content/02.introduction.md

```diff
@@ -1,29 +1,25 @@
 ## Introduction
+
+Scientific communication has been around since the beginning of science where the main form of communication was through scientific meetings [@raw:smith2006trouble].
+Communicating through scientific journals started back in the 17th century [@raw:smith2006trouble] but didn't pick up until the 1960s where National Institutes of Health (NIH) created an exchange network that allowed scientists to share unreferred manuscripts with one another [@doi:10.1371/journal.pbio.2003995].

```

Comment:
I'm confused here. Is the argument that the NIH's unrefereed manuscript exchange network is related to when communicating through journals picked up?